docs/scos/dev/troubleshooting/troubleshooting-spryker-in-cloud-issues/missing-migration-files.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Troubleshooting Spryker in Cloud issues. Missing migration files on production environment 
+description: Troubleshoot issues you might encounter when you have your Spryker-based project in Cloud. Missing migration files on production environment
+last_updated: Jun 9, 2022
+template: ?
+originalLink: ?
+originalArticleId: ?
+redirect_from:
+---
+
+Missign migration files on production environments. 
+
+## Note
+It is recomended to push all the migrations to a repository as a go-live preparation and not during the development (to simplify the process of development). 
+
+## Solution
+
+1. Check if the files are not under ignore (f.e. .dockerignore) and added to the version control system (f.e. github).
+2. Check that during the deployment the files are not removed (f.e. check if during the deployment you are not using the ```console propel:migration:delete``` command that removes migrations).
```

Comment:
"Check that during the deployment the files are not removed"

How/where to check that? I know the answer, but not sure it is so obvious for the customer. You could mention config/install - or better reference corresp. pages. I was able to find [this one](https://docs.spryker.com/docs/cloud/dev/spryker-cloud-commerce-os/configuring-deployment-pipelines/deployment-pipelines.html) and [this one](https://docs.spryker.com/docs/cloud/dev/spryker-cloud-commerce-os/configuring-deployment-pipelines/customizing-deployment-pipelines.html)

Reply:
Please add it